src/Utilities/Refactoring.VisualBasic/Extensions/StatementSyntaxExtensions.vb

```diff
@@ -6,7 +6,9 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
 Namespace Global.Analyzer.Utilities.Extensions
 
+#Disable Warning IDE1006 ' Naming Styles - False positive (missing prefix 'I')
```

Comment:
😕 Where is this rule coming from? I don't see it in .editorconfig.

Reply:
https://github.com/dotnet/roslyn-analyzers/blob/dba3183193df36d666fc48c0eac0d2d640d0041d/.editorconfig#L188

Reply:
File bug for it in Roslyn?

Reply:
@Youssef1313 bug for what? This is the design - we are explictly requesting all Style analyzers coming from CodeStyle package to be warning diagnostics. This way when we move to a newer package, we automatically get build warnings for newly added analyzers. This basically allows us to switch from opt-in mode to opt-out mode for analyzer warnings from this package.

Reply:
@mavasani Isn't it false positive? Modules doesn't have to be prefixed with `I` (unlike interfaces).

Reply:
@Youssef1313 Ah yes, sorry I misunderstood your suggestion. It was not for the fact that it is escalated to the warning, but the fact that the analyzer itself is reporting a false positive. I'll file a Roslyn bug.

Reply:
Alright, so seems like this is not an analyzer issue - I had introduced this rule in Tools Options during my testing and forgot to delete it. I will revert these suppressions.

Reply:
Reverted now.